sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/LocalDocumentStorage.java

```diff
@@ -195,6 +195,12 @@ private static SQLiteQueryBuilder getPartitionAndDocumentIdQueryBuilder() {
         return write(document, writeOptions, Constants.PENDING_OPERATION_REPLACE_VALUE);
     }
 
+    boolean writeDelete(String partition, String documentId) {
```

Comment:
What's the difference between writeDelete and delete in the same class? This class methods really need some Javadoc as this is very confusing when we don't know the code base here.

Reply:
Delete removes the document from the cache, writeDelete adds DELETE to the pending operations column. I’ll add javadoc to clarify the difference.